cockroachcloud/free-faqs.md

```diff
@@ -0,0 +1,105 @@
+- [curbshop.online](https://devpost.com/software/curbshop-online)
+
+### What are the limitations of CockroachCloud Free?
+
+CockroachCloud Free is currently in beta and as such, it is not suitable for heavy production use. Specifically, there are capabilities we are still working on enabling, such as the ability to enable backups, to import data, and no-downtime upgrades to a paid tier. If you want to use any of these capabilities, try a [30-day trial of CockroachCloud](quickstart-trial-cluster.html).
+
+### Can I use a specific region in AWS or GCP?
+
+At this time, CockroachCloud Free is only available in the GCP `us-central-1` region.
+
+### How do I connect to my cluster?
+
+To connect to a cluster, download the CA certificate, and then generate a connection string or parameters. You can use this information to connect to your cluster through the CockroachDB SQL client or a Postgres-compatible driver or ORM. For more details, see [Connect to Your CockroachCloud Cluster](connect-to-your-cluster.html).
+
+## Beta release
+
+### Why is CockroachCloud Free in beta?
+
+CockroachCloud Free is in beta while we work on adding core features like [import](../v20.2/import.html) and [backups](backups-page.html).
+
+### Where can I submit feedback or bugs on the beta?
+
+You can submit feedback or log any bugs you find through [this survey](https://cockroachlabs.typeform.com/to/gvCcF14q).
+
+## Security
+
+### Is my cluster secure?
+
+We use separate certificate authorities for each cluster, and all connections to the cluster over the internet use TLS 1.2.
+
+### Is encryption-at-rest enabled on CockroachCloud Free?
+
+Yes. All data on CockroachCloud is encrypted-at-rest using the tools provided by the cloud provider that your cluster is running in.
+
+- Data stored in clusters running in GCP are encrypted-at-rest using [persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption).
+- Data stored in clusters running in AWS are encrypted-at-rest using [EBS encryption-at-rest](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html).
+
+Because we are relying on the cloud provider's encryption implementation (as noted above), we do not enable CockroachDB's [internal implementation of encryption-at-rest](../v20.2/encryption.html#encryption-at-rest-enterprise). This means that encryption will appear to be disabled in the [DB Console](../v20.2/ui-overview.html), since it is unaware of cloud provider encryption.
+
+### Is my cluster isolated? Does it share resources with any other clusters?
+
+CockroachCloud Free is a multi-tenant offering and resources are shared between clusters.
+
+## Cluster maintenance
+
+### How do I add nodes?
+
+We have eliminated the concept of nodes for CockroachCloud Free. However, there is an upper limit of usage of up to 1 vCPU and 5GB storage per free cluster. If you exceed this limit, you can create a cluster using the paid version of [CockroachCloud](create-your-cluster.html).
```

Comment:
As a reader I would prefer to read something more direct like "You can not add nodes to your CockroachCloud Free instance; it is limited to 1vCPU ....   If you want a more powerful cluster with multiple nodes, use the paid version ..."

Because actually we have not eliminated the concept of nodes, because we are talking about nodes